src/Microsoft.DotNet.Build.Tasks/project.json

```diff
@@ -18,6 +18,7 @@
     "NuGet.Versioning": "4.3.0-preview2-4095",
     "System.Diagnostics.TraceSource": "4.0.0",
     "System.Dynamic.Runtime": "4.0.11",
+    "System.IO.UnmanagedMemoryStream": "4.3.0",
```

Comment:
I wonder if this pulls up the versions of the other packages to the 4.3 set? I don't know if that will hurt anything but something to keep in mind if something goes wrong.